src/main/java/com/commercetools/sync/categories/CategorySync.java

```diff
@@ -144,7 +145,7 @@ public CategorySync(@Nonnull final CategorySyncOptions syncOptions) {
      */
     @Override
     protected CompletionStage<CategorySyncStatistics> processBatch(@Nonnull final List<CategoryDraft> categoryDrafts) {
```

Comment:
i'm confused how `processBatch` should be executed: concurrently or sequentially? If there are multiple batches, should they be processed one-by-one, or simultaneously?

Reply:
They should be processed one-by-one. It is not meant for batches to be processed concurrently.

Reply:
If so, i'm confused where in the method body/execution workflow concurrency occurs?

Reply:
I added now JavaDocs on the concurrent/non-concurrent maps and sets.